presto-docs/src/main/sphinx/connector/base-arrow-flight.rst

```diff
@@ -0,0 +1,106 @@
+
+2. ``ArrowAbstractFlightRequest.java``
+Implement this class to define the request data, including the data source type, connection properties, the number of partitions and other data required to interact with database.
+
+3. ``ArrowAbstractMetadata.java``
+To retrieve metadata (schema and table information), implement the abstract methods in the ArrowAbstractMetadata class.
+
+4. ``ArrowAbstractSplitManager.java``
+Extend the ArrowAbstractSplitManager class to implement the Arrow flight request, defining the Arrow split.
+
+5. ``ArrowPlugin.java``
+Register your connector name by extending the ArrowPlugin class.
+
+Configuration
+-------------
+To configure the Arrow connector, create a catalog file
+in ``etc/catalog`` named, for example, ``arrowmariadb.properties``, to
+mount the Arrow-flight connector as the ``arrowmariadb`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+
+.. code-block:: none
+
+
+        connector.name=<CONNECTOR_NAME> 
+
+        arrow-flight.server=<SERVER_ENDPOINT>
+        arrow-flight.server.port=<SERVER_PORT>
+
+        data-source.name=<DATASOURCE_NAME>
+        data-source.host=<HOST>
+        data-source.database=<DATABASE_NAME>
+        data-source.username=<USERNAME>
+        data-source.password=<PASSWORD>
+        data-source.port=<PORT>
+        data-source.ssl=<TRUE/FALSE>
+
+        Note : You can add other properties that are required for your Flight server to connect.
+
+========================================== ==============================================================
+Property Name                               Description
+========================================== ==============================================================
+``arrow-flight.server``                     Endpoint of arrow-flight server
+``arrow-flight.server.port``                Flight server port
+``data-source.name``                        Datasource name
+``data-source.host``                        Hostname of the database
+``data-source.database``                    Database name
+``data-source.username``                    Username of database
+``data-source.password``                    Password of database
+``data-source.port``                        Database port
+``data-source.ssl``                         Enable SSL for datasource True/False
+``arrow-flight.server-ssl-certificate``     Pass ssl certificate
+``arrow-flight.server.verify``              To verify server
+``arrow-flight.server-ssl-enabled``         Port is ssl enabled
+========================================== ==============================================================
+
+Querying Arrow-Flight
+---------------------
+
+The Arrow-Flight connector provides schema for every supported *databases*.
```

Comment:
```suggestion
The Arrow-Flight connector provides schema for each supported *database*.
```
Is this each _supported_ database, or each _configured_ database (that has a catalog file in ``etc/catalog``?